cores/esp8266/core_esp8266_i2s.c

```diff
@@ -331,6 +331,45 @@ bool i2s_write_lr(int16_t left, int16_t right){
   return i2s_write_sample(sample);
 }
 
+#define MIN(a,b) (a<b?a:b)
```

Comment:
This is only used the one time later on, can we dispense with the macro?  MIN() as a macro is always dangerous, but obviously std::min() isn't an option for .C code.

Reply:
I can inline that macro in the code, is that fine?